pkg/proxy/ipvs/ipset_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ipvs
+
+import (
+	"testing"
+)
+
+func TestCheckIPSetVersion(t *testing.T) {
+	testCases := []struct {
+		vstring string
+		valid   bool
+	}{
+		{"4.0", false},
```

Comment:
how about 4.1.3 or something like that?

Reply:
4.1.3 is invalid, since it's less than  6.0

7 is invalid, since it's not a valid version string.

6.1.3 is valid...

I added some more test cases. PTAL. Thanks!